k2/csrc/algorithms.h

```diff
@@ -1,13 +1,18 @@
-// k2/csrc/cuda/algorithms.h
-
-// Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey)
-
-// See ../../LICENSE for clarification regarding multiple authors
-
-#ifndef K2_CSRC_CUDA_ALGORITHMS_H_
-#define K2_CSRC_CUDA_ALGORITHMS_H_
-
-#include "k2/csrc/cuda/array.h"
+/**
+ * @brief
```

Comment:
please follow the comment you posted in <https://github.com/danpovey/k2/issues/118>
- `@file` is missing
- A sentence  describing the file briefly is replaced with only a single word. Please use
a meaningful sentence instead of a word.

Reply:
The @file is not official requirements. I regret to keep it, as the filename may change, and it's useless for doxygen, as it is able to add it auto.. 

About the brief, I have to make the filename as it, as I cannot describe each one.

Reply:
A single word for `@brief` makes nonsense.